src/main/groovy/com/avast/gradle/dockercompose/ComposeSettings.groovy

```diff
@@ -179,6 +186,14 @@ class ComposeSettings {
         captureContainersOutputToFile = file
     }
 
+    void setComposeLogFile(CharSequence path) {
```

Comment:
I would prefer to remove this setter and getter, like in case of `captureContainersOutputToFile`.

Reply:
I'm not sure what you mean.  `captureContainersOutputToFile` does have two setters, one for string and one for file.  I think this is convenient.